### PR TITLE
Feel blue

### PR DESCRIPTION
**Pull request by denesnori**
Unify styling.

### CODE REVIEW

**Review by jbesraa — APPROVED** [review]

**Comment by des-des** [comment]
@denesnori commit messages missing commit numbers !!!!!!

**Comment by des-des** [comment]
No issue number in pull desc!!

**Comment by des-des** [comment]
#13

File: css/cards.css [hunk 1]
```diff
@@ -11,25 +26,50 @@ header, main, footer {
...
   max-width: 65% !important;
+  height: 100%;
 }
 
-.card-image-duo{
+.try{
   display:block;
-  max-width:17.5% !important;
+  max-width:35%;
+  /*border:solid green 1px;*/
```
Comment (des-des) [inline css/cards.css]: :fire:

File: js/cards.js [hunk 2]
```diff
@@ -116,7 +116,7 @@ $(document).ready(function(){
     let newUrl= url.split("/")
     let key=newUrl.pop();
     let num=key.split("=")[1]||1;
-    if (num===1) {
+    if (num==1) {
```
Comment (des-des) [inline js/cards.js]: should be `===`. If the types are wrong, you need to be explicit, you can change `1` to `"1"` or use parseInt

**Review by des-des — CHANGES_REQUESTED** [review]

File: js/cards.js (hunk 2 above)
Comment (denesnori) [reply js/cards.js]: OK :) I'll do it

**Comment by denesnori** [comment]
ups, I 've seen your other comment. I referencd issue 21 and 22 in some of the commit....but not in all...beacuse I though that it would indicate this pull requests belong to thos issues, but I felt pointless to reference all 10-20 pushes on this branch. Should I reference the issues every single time?

**Comment by des-des** [comment]
@denesnori Yes, each commit sound have an issue

**Comment by des-des** [comment]
@denesnori are you sure your fixes work? Have you tested them? 6 === '6' is false..?

**Comment by denesnori** [comment]
Working from new branch with CSS changes

**Comment by des-des** [comment]
@denesnori The idea is not to merge code into master with obvs bugs.. As I review each pull in isolation,

**Comment by des-des** [comment]
:( sorry!

**Comment by des-des** [comment]
was about to say am happy to merge a long as you understand!

**Pull request closed without merging**